cmd/estuary-shuttle/main.go

```diff
@@ -51,8 +53,6 @@ import (
 	"github.com/whyrusleeping/memo"
 )
 
-var Tracer = otel.Tracer("shuttle")
```

Comment:
this moved onto the Shuttle struture so they can be viewed independently, i.e. which shuttle produced which trace.